webapp/backend/pkg/database/scrutiny_repository_device_smart_attributes.go

```diff
@@ -30,14 +31,14 @@ func (sr *scrutinyRepository) SaveSmartAttributes(ctx context.Context, wwn strin
 }
 
 // GetSmartAttributeHistory MUST return in sorted order, where newest entries are at the beginning of the list, and oldest are at the end.
-func (sr *scrutinyRepository) GetSmartAttributeHistory(ctx context.Context, wwn string, durationKey string, attributes []string) ([]measurements.Smart, error) {
+func (sr *scrutinyRepository) GetSmartAttributeHistory(ctx context.Context, wwn string, durationKey string, n int, offset int, attributes []string) ([]measurements.Smart, error) {
```

Comment:
```suggestion
func (sr *scrutinyRepository) GetSmartAttributeHistory(ctx context.Context, wwn string, durationKey string, selectEntries int, selectEntriesOffset int, attributes []string) ([]measurements.Smart, error) {
```

Reply:
a comment would be helpful here. as well, describing how the selectEntries* keys are used -- and the fact that `selectEntriesOffset` is only relevant when `selectEntries` is >0. 

We could technically make `selectEntries` a `*int` type, to differentiate between 0 and "set", but that makes parameter passing kind-of annoying -- and it's unnecessary, since selecting 0 entries has no meaning.
I think we just need a big comment block here.